src/pages/portfolio-page/index.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Container, Box, makeStyles, Card, useTheme, useMediaQuery } from '@material-ui/core';
+
+import { DetailsCard } from '../../components/DetailsCard';
+import { SectionTitle } from '../../components/SectionTitle';
+import { PortfolioCard } from '../../components/PortfolioCard';
+
+import { projectData, userData } from './data';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    padding: 0,
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      minHeight: '100vh',
+      padding: theme.spacing(8, 2, 0, 2),
+    },
+  },
+  aside: {
+    display: 'none',
+    [theme.breakpoints.up('lg')]: {
+      display: 'block',
```

Comment:
Box is a div, so it has `display: block` by default

Reply:
But it is set to `none` so I have to set it back to `block`. Is there a better way to do it?